src/main/kotlin/lotto/LottoNumber.kt

```diff
@@ -0,0 +1,21 @@
+package lotto
+
+@JvmInline
+value class LottoNumber private constructor(val value: Int) {
+    init {
+        require(value in MIN_NUMBER..MAX_NUMBER) {
+            "로또 번호는 $MIN_NUMBER~$MAX_NUMBER 사이여야 합니다."
+        }
+    }
+
+    companion object {
+        private const val MIN_NUMBER = 1
+        private const val MAX_NUMBER = 45
+
+        val cache = (MIN_NUMBER..MAX_NUMBER).associateWith { LottoNumber(it) }
```

Comment:
```suggestion
        val CACHE = (MIN_NUMBER..MAX_NUMBER).associateWith { LottoNumber(it) }
```
상수의 네이밍은 기본적으로 대문자를 사용합니다.

Reply:
앗 맞네요 이 부분을 놓쳤군요